akka-stream/src/main/mima-filters/2.5.8.backwards.excludes

```diff
@@ -45,4 +45,11 @@ ProblemFilters.exclude[MissingClassProblem]("akka.stream.stage.GraphStageLogic$C
 ProblemFilters.exclude[FinalClassProblem]("akka.stream.stage.GraphStageLogic$ConcurrentAsyncCallback$Pending")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffLogic.onCompleteOrFailure")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffLogic.this")
-ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffSource.this")
\ No newline at end of file
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffSource.this")
+# Add maxRestarts to RestartSource, Sink and Flow https://github.com/akka/akka/pull/24137
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartSource.onFailuresWithBackoff")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartSource.withBackoff")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartSink.withBackoff")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffSink.this")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffFlow.this")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartFlow.withBackoff")
```

Comment:
this is adding too many filters that are not needed. It should be enough with
```
ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffSink.this")
ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.RestartWithBackoffFlow.this")
```

apart from that this PR looks good and can be merged